src/app/team/serviceForm/tgncClothing/ServiceTgncClothing.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { compose, withProps } from 'recompose';
+import { getService, getServiceTgncClothing, getServiceId } from '../../../../selectors/service';
+import { selectLocationError } from '../../../../selectors/location';
+import * as actions from '../../../../actions';
+import { Form } from '../../../../components/form';
+import Edit from './Edit';
+import View from './View';
+
+const FormComponent = compose(withProps({
+  ViewComponent: View,
+  EditComponent: Edit,
+}))(props => <Form {...props} />);
+
+const mapStateToProps = (state, ownProps) => ({
+  resourceData: getService(state, ownProps),
+  value: getServiceTgncClothing(state, ownProps),
+  id: getServiceId(ownProps),
+  resourceLoadError: selectLocationError(state, ownProps),
+});
+
+const mapDispatchToProps = (dispatch, ownProps) => ({
+  fetchResourceData: bindActionCreators(actions.getLocation, dispatch),
+  updateValue: (tgncClothing, serviceId, metaDataSection, fieldName) =>
```

Comment:
Is it possible there's some problem with this update? I was trying it locally, and got an error whenever I tried to save any value here (something about "id of undefined" in services/service.js, on the api side).
But maybe that's just a local issue, or maybe it's one of those you've already fixed.

Reply:
I remember on local i was missing a field, this must be it.

Is it possible that the migration didn't add the `tgncClothing` field on staging?

Reply:
Just checked, and Staging does have `tgncClothing` in taxonomy_specific_attributes.

Reply:
I have two DB dumps on my local files, i tried both and there is one where the `tgncClothing` field does not exist, can you confirm that on your local setup that field exists?

Reply:
My local DB doesn't have `tgncClothing`, though I'm pretty sure I saw this problem when working with the Staging DB, not my local one.

But if you're confident that's the only cause for this error, then that's OK.
We can merge this, deploy to Staging, and hope it works.